libs/datasets/manual_filter.py

```diff
@@ -96,3 +100,70 @@ def run(dataset: timeseries.MultiRegionDataset, config=CONFIG) -> timeseries.Mul
         dataset = filtered_dataset.append_regions(passed_dataset)
 
     return dataset
+
+
+# Possible values from https://github.com/covid-projections/covid-projections/blob/develop/src/cms-content/region-overrides/region-overrides.json
+_METRIC_TO_FIELDS = {
+    "metrics.caseDensity": common_fields.FIELD_GROUP_TO_LIST_FIELDS[FieldGroup.CASES_DEATHS],
```

Comment:
I think we probably want this to just be cases.  Or we could rename "metrics.caseDensity" to something else (see below), but in general I think we block cases way more than deaths and probably want to treat them separate.

Reply:
Changed.